### ISSUE TITLE
Add Google Analytics code to Online Tool and Form Detail CTA Links

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#17898]
## Issue Description
The Analytics team provided tagging information to be added into the Find a VA Form search results so that clicks to forms detail pages and online tools can be tracked.

---
## Tasks
- [ ] Add Google Analytics code provided in issue https://github.com/department-of-veterans-affairs/va.gov-team/issues/17273 to the search results on Find a VA Form. 

## Acceptance Criteria
- [ ] GA code is added and peer reviewed
- [ ] Clicks on Go To Online Tool button and links to form detail pages can successfully be tracked in Google Analytics on staging
- [ ] Changes are reviewed and approved by Product Owner and Product Manager prior to Prod deployment

**Comment by zacharymorel** [comment]
@MarciMcGuireGCIO how do you feel about removing "and sorting links" from the AC?
Since we are still waiting on the Sort feature a11y review of the design, I think narrowing down this user story to apply to  what is already done can enable me to start on it. 

Once the Sorting functionality comes, I think we can add the GA tags as apart of that scope of that work.

**Comment by MarciMcGuireGCIO** [comment]
> @MarciMcGuireGCIO how do you feel about removing "and sorting links" from the AC?
> Since we are still waiting on the Sort feature a11y review of the design, I think narrowing down this user story to apply to what is already done can enable me to start on it.
> 
> Once the Sorting functionality comes, I think we can add the GA tags as apart of that scope of that work.

Great minds... 😆 I've already pulled that part out and added it to this ➡️ https://github.com/department-of-veterans-affairs/va.gov-team/issues/17901
